### ISSUE TITLE
Tag Bridged Channels

### ISSUE THREAD

**Issue by niebloomj** [SchildiChat/SchildiChat-android#230]
**Is your feature request related to a problem? Please describe.**
I want to be able to reference another room. Say I'm messaging someone on slack, I want to be able to say "the better place to ask this would be in #otherchannel" 

**Describe the solution you'd like**
I would like the client to be able to popup with a list of all rooms and on my end the room should link to the bridge room but for the recipient it links to the real platforms linked channel. 

**Describe alternatives you've considered**
Not sure. 

**Additional context**
Currently, as you can see in my screenshot, it only lets you tag matrix non bridged rooms. ![Screenshot_20240228-122307.png](https://github.com/SchildiChat/SchildiChat-android/assets/5156403/7429473a-b686-4287-9d50-2f1d3703a4a4)

**Comment by SpiritCroc** [comment]
So the client would need to understand link formats of all available bridges? I'd say that's out of scope for SchildiChat, which is a Matrix-first client. Currently, it doesn't even know anything about bridges you use. (I could see myself adding some network icon for some known bridges at some point, maybe, but that's about it for bridge-specific functionality)

**Comment by niebloomj** [comment]
I don't think so. The client would just need to show the link to the relevant channel within matrix. It would be the bridge's job to convert the link to a matrix channel to a matching channel within the bridge like a slack channel or whatsapp group. From the client side its all matrix.

**Comment by SpiritCroc** [comment]
Ah, I thought you were saying like inter-bridges, e.g. if one messages you on tg and you say, hey, signal would be better. Bridge-translation wouldn't work here since bridges don't talk to each other.
Well, for bridge-unspecific issues I suggest writing an issue title without "bridge" in it.

> Currently, as you can see in my screenshot, it only lets you tag matrix non bridged rooms. 

It's only suggesting you rooms with a public address, i.e. room addresses starting with a `#` - which makes sense to me, since these are more commonly useful when being linked, as they're more likely to be public joinable. Linking private rooms can be done by opening said private rooms and clicking the share button in the room settings. Linking private rooms shouldn't be a too common use case to justify spending time on implementing auto completions to me.
